### PR TITLE
Cat col speed improvements

### PR DESCRIPTION
**Pull request by Colin-Hanley**

### CODE REVIEW

**Review by chris-pickford — CHANGES_REQUESTED** [review]
all good, just a couple of minor things

File: datadayessentials/data_transformation/_transformers.py [hunk 1]
```diff
@@ -437,6 +438,68 @@ def process(self, df_in: pd.DataFrame):
...
+        # Convert the numerical series to numeric if required
+        if force_numeric:
+            numerical_series = pd.to_numeric(numerical_series, errors="coerce")
+            numerical_series.name = f"{value.name}_num"
+
+        return cat_series, numerical_series
+
+    def process(self, df_in: pd.DataFrame) -> pd.DataFrame:
+        for column in self.categorical_columns_to_split:
+            if 'QCB' in column and column in df_in.columns:
```
[the remark this answers is not in the document]
Comment (Colin-Hanley) [reply datadayessentials/data_transformation/_transformers.py]: I had the same thought last night, but I now agree that I made wrong choice. I've extracted the QCB filter outside of the function.

**Review by whitew1994WW — APPROVED** [review]
No problems here, nice job 👍

File: datadayessentials/data_transformation/_transformers.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Colin-Hanley) [reply datadayessentials/data_transformation/_transformers.py]: would it not be quicker to send in the correct columns rather than have a loop checking ?

**Review by chris-pickford — APPROVED** [review]
beautiful

**Pull request merged**